juicy-html.html

```diff
@@ -7,18 +7,75 @@
+            );
+        }
+        /**
+         * Translate content tag changes to href and html attributes
+         * @param {JuicyHTMLElement} element to set attibutes on
+         * @param {String} value value of content attr
+         */
+        function forwardContentValue(element, value){
+            if (value && (value.indexOf('/') === 0 || value.indexOf('./') === 0 || value.indexOf('../') === 0)) {
+                //value is a URL, load the partial from external file
+                element.removeAttribute('html');
+                element.setAttribute('href', value);
+            } else if (value === null) {
+                element.removeAttribute('html');
+                element.removeAttribute('href');
+            } else {
+                //value is HTML code, insert the partial from the string
+                element.removeAttribute('href');
+                element.setAttribute('html', value);
+            }
+        }
+
         var JuicyHTMLPrototype = Object.create((HTMLTemplateElement || HTMLElement).prototype);
         var isSafari = navigator.vendor && navigator.vendor.indexOf("Apple") > -1 && navigator.userAgent && !navigator.userAgent.match("CriOS");
 
         JuicyHTMLPrototype.createdCallback = function() {
-            var model = null;
-            Object.defineProperty(this, "model", {
-                set: function(newValue) {
-                    model = newValue;
-                    this.attachModel(newValue);
+            var model = null; // XXX(tomalec): || this.model ?
+            Object.defineProperties(this, {
+                'model': {
+                    set: function(newValue) {
+                        model = newValue;
+                        this.attachModel(newValue);
+                    },
+                    get: function() {
+                        return model;
+                    }
                 },
-                get: function() {
-                    return model;
+                'href': {
+                    set: function(newValue) {
+                        if(newValue === null){
+                            this.removeAttribute('href');
+                        } else {
+                            this.setAttribute('href', newValue);
+                        }
+                    },
+                    get: function() {
+                        return this.getAttribute('href');
+                    }
+                },
+                'html': {
+                    set: function(newValue) {
+                        if(newValue === null){
+                            this.removeAttribute('html');
+                        } else {
+                            this.setAttribute('html', newValue);
+                        }
+                    },
+                    get: function() {
+                        return this.getAttribute('html');
+                    }
                 }
             });
         };
```

Comment:
I think that you get better performance by forwarding attribute setters to property setters, not property setters to attribute setters as done above.

Reply:
Why do you think so?

Also, this is the way suggested at 
spec https://www.w3.org/TR/custom-elements/#custom-elements-autonomous-example
and tutorials 
https://developers.google.com/web/fundamentals/architecture/building-components/customelements

Reply:
Sorry, ignore this, that doesn't make any sense :)